### ISSUE TITLE
[Live Preview MultiBrowser] Live highlighting not working for less files

### ISSUE THREAD

**Issue by mkurz** [adobe/brackets#10635]
When setting `"livedev.multibrowser": true` live highlighting for `.less` files does not work, neither in latest Chrome or Firefox. However it does work normally for the generated `.css` files. (I am using [`LESS AutoCompile v1.1.7`](https://github.com/jdiehl/brackets-less-autocompile))
When setting ``"livedev.multibrowser"` back to `false` live highlighting for `.less` files is working normal again.
So obviously this is a bug within the new multibrowser feature.

Brackets Version:

    Release 1.1 experimental build 1.1.0-15558 (release ea908cae5) 
    build timestamp: Thu Dec 18 2014 10:46:04 GMT-0800

OS:

    Ubuntu 12.04.5 LTS x86_64

**Comment by le717** [comment]
Not saying I'm correct in this assessment, but it looks as if the issue lies in the `_classForDocument()` function in LiveDevMultiBrowser.js. I'm thinking a quick condition change to also call `FileUtils.isCSSPreprocessorFile()` may do the trick.

**Comment by peterflynn** [comment]
Good catch!  It looks like a whole new LiveCSSPreprocessorDocument module would have to be implemented also, so it's probably not a totally trivial fix...

CC @sebaslv & @busykai
